detect_secrets/plugins/common/filters.py

```diff
@@ -80,13 +80,34 @@ def is_sequential_string(secret, *args):
     return False
 
 
-ALL_FALSE_POSITIVE_HEURISTICS = (
+# This only finds UUIDs which only have lowercase characters.
+_UUID_REGEX = re.compile(r'[a-f0-9]{8}\-[a-f0-9]{4}\-[a-f0-9]{4}\-[a-f0-9]{4}\-[a-f0-9]{12}')
+
+
+def is_potential_uuid(secret, *args):
+    """
+    Determines if a potential secret contains any UUIDs.
+
+    :type secret: str
+
+    :rtype: bool
+    Returns True if the string has a UUID, false otherwise.
+    """
+
+    # Using a regex to find strings that look like false-positives
+    # will find us more false-positives than if we just tried validate
+    # the input string as a UUID (for example, if the string has a prefix
+    # or suffix).
+    return len(_UUID_REGEX.findall(secret.lower())) > 0
```

Comment:
I would maybe do `bool(_UUID_REGEX.search(secret))` since `findall` might be more than we need

I realize we either do `.lower()` or `.upper()` in each filter function (`.lower` in this func and `is_found_with_aho_corasick`, then `.upper` in `is_sequential_string`, maybe we can DRY that up in `is_false_positive`.

Reply:
I just pushed a change to make these regexes case insensitive :)

Good point about the `search` vs `findall` distinction, I will do that